.buildkite/bk.integration.pipeline.yml

```diff
@@ -8,19 +8,19 @@ env:
   ASDF_TERRAFORM_VERSION: 1.9.2
 
 steps:
-  - label: "Integration tests: packaging"
-    key: "package-it"
-    command: ".buildkite/scripts/steps/integration-package.sh"
-    artifact_paths:
-      - build/distributions/**
-    agents:
-      provider: "gcp"
-      machineType: "n1-standard-8"
+  # - label: "Integration tests: packaging"
```

Comment:
Those are changes to the pipeline that @pazone had added in a previous reincarnation of this PR for testing purposes... I isolated them in their own commit but at this point I think we can drop such test commit

Reply:
It was done for testing purposes. Since we are testing it in another branch we can revert all changes in `bk.integration.pipeline` and leave only the changes related to filtering